rosdep/base.yaml

```
@@ -750,6 +750,12 @@ daemontools:
   nixos: [daemontools]
   openembedded: [daemontools@meta-oe]
   ubuntu: [daemontools]
+dcraw:
+  arch: [dcraw]
+  debian: [dcraw]
+  fedora: [dcraw]
+  gentoo: [media-gfx/dcraw]
+  ubuntu: [dcraw]
```

Checks are suggesting an existing OpenSUSE package, and looks like we encourage OpenSUSE packages to be added now.
```suggestion
  opensuse: [dcraw]
  ubuntu: [dcraw]
```